#There are some params which will be loaded
    
max_acceleration: 1.5
max_deceleration: -1.5

max_brake_value: 0.0
min_brake_value: 0.0

max_speed_backword: 1.0
max_speed_forward: 3.0
min_speed_forward: 0.0

max_steer_angle: 0.4
min_steer_angle: 0.4

max_steer_value: 660
min_steer_value: -660
